roles/openshift_node/tasks/apply_machine_config.yml

```diff
@@ -0,0 +1,90 @@
+---
+- name: Create temp directory
+  tempfile:
+    state: directory
+  register: temp_dir
+
+- name: Get worker machine config name
+  command: >
+    oc get machineconfigpool worker
```

Comment:
This should fetch MC name from node annotations maybe?

Reply:
I believe we need to still grab the MC from the machineconfigpool (which is what the MCS does anyway)

Reply:
also, as discussed on Slack, let's do:

```
oc get -o json machineconfig $(oc get machineconfigpool worker -o jsonpath='{.status.configuration.name}') | jq '.spec.config'
```

To obtain the Ignition config directly for the worker and use that in MCD --once-from with Ignition (as we're already used to). So the only change would be where we grab the Ingnition for the pool (MC vs MCS)

Reply:
I'll make these changes and update.

Reply:
Updated.